src/components/ColorSwitch/ColorSwitch.tsx

```diff
@@ -8,14 +8,30 @@ import styles from './colorswitch.module.css';
 
 interface ColorSwitchProps {
     className?: string;
+    isDarkMode: boolean;
+    onThemeChange: (isDarkMode: boolean) => void;
 }
```

Comment:
```suggestion
interface ColorSwitchProps {
    isDarkMode: boolean;
    onThemeChange: (isDarkMode: boolean) => void;
     className?: string;
}
```

optional parameters should be always in the end of the interface